pkg/osquery/table/touchid_system_darwin.go

```diff
@@ -0,0 +1,88 @@
+package table
+
+import (
+	"bytes"
+	"context"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/pkg/errors"
+
+	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+)
+
+func TouchIDSystemConfig(client *osquery.ExtensionManagerClient, logger log.Logger) *table.Plugin {
+	t := &touchIDSystemConfigTable{
+		client: client,
+		logger: logger,
+	}
+	columns := []table.ColumnDefinition{
+		table.IntegerColumn("touchid_compatible"),
+		table.TextColumn("secure_enclave_cpu"),
+		table.IntegerColumn("touchid_enabled"),
+		table.IntegerColumn("touchid_unlock"),
+	}
+
+	return table.NewPlugin("touchid_system_config", columns, t.generate)
```

Comment:
Please prepend all table names with `kolide_` so we don't accidentally conflict with other plugins